ui/admin/app/templates/onboarding/success.hbs

```diff
@@ -0,0 +1,90 @@
+{{page-title (t 'titles.success')}}
+<Rose::Layout::Centered class='create-resources'>
+  <Rose::Frame>
+    <:header>
+      <h2>{{t 'onboarding.success.titles.connect'}}</h2>
+    </:header>
+    <:body>
+      {{! We have created }}
+      <div class='create-resources-item'>
+        <span class='create-resources-item-header'>
+          <Rose::Icon @name='flight-icons/svg/check-circle-16' />
+          <h3>{{t 'onboarding.success.titles.creation-success'}}</h3>
+        </span>
+        <p>{{t 'onboarding.success.descriptions.role'}}</p>
+      </div>
+
+      {{! Donwload Desktop client }}
+      <div class='create-resources-item'>
+        <span class='create-resources-item-header'>
+          <Rose::Icon @name='logo-app-background' @size='extra-large' />
```

Comment:
Can we use the `<Hds::IconTile />` here?

Reply:
Awesome catch and even better suggestion! you got it 😉